src/__experimental__/components/search/search.component.js

```diff
@@ -44,13 +43,16 @@ const Search = ({
     }
 
     return '';
-  }, [isFocused, searchIsActive, searchValue, threshold, searchButton]);
+  }, [searchValue.length, isFocused, searchIsActive, value, isControlled, threshold, searchButton]);
```

Comment:
seems a bit odd to have `searchValue.length` but only `value` in the dependency of this `useMemo` callback, I think adding `value.length` instead of `value` will be fine here

Reply:
Yea, code formatter has changed this one :D